Dockerfile

```diff
@@ -25,6 +25,16 @@ WORKDIR /app/
 ENV PYTHONUNBUFFERED=1 \
     PYTHONPATH=/app/
 
+RUN apt-get -y update && apt-get install -y curl gnupg
+
+# Install Cloud SDK: https://cloud.google.com/sdk/docs/quickstart-debian-ubuntu
+RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | \
```

Comment:
Are you sure this is needed? I don't see it being used by anything further down, and I don't think the Python module depends on it.

Reply:
It might not be needed, but it's nice to have access to the command-line tools like `gsutil` and `bq` for management. I'll drop this commit after everything is tested, if it's not needed.

Reply:
Sounds good to me!

Reply:
Ended up dropping this, because none of the management needs to be done from the CLI.